spec/factories/assessments.rb

```diff
@@ -55,6 +56,11 @@
       end
     end
 
+    trait :decline do
```

Comment:
I think it would be good if this was defined directly below the award trait, they are very similar and it’ll make it easier to find the traits when looking through this file.